src/rgw/CMakeLists.txt

```diff
@@ -397,3 +397,8 @@ if(WITH_TESTS)
     ceph_rgw_multiparser
     DESTINATION bin)
 endif(WITH_TESTS)
+
+configure_file(${CMAKE_SOURCE_DIR}/src/rgw/rgw-orphan-list.in
```

Comment:
i don't think we need to prepare `rgw-orphan-list.in` as a `config_file` as it's a ready-to-use script without replacing the substitution variables in it with the variables in cmake file. instead, i'd suggest
```cmake
install(PROGRAMS rgw-orphan-list
  DESTINATION bin)
```

in which i assume that `src/rgw/rgw-orphan-list.in` is renamed to `src/rgw/rgw-orphan-list`.